README.md

```diff
@@ -1 +1,44 @@
-# Math-Magicians
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math-Magicians
+
+> Description the project.
+
+> This is the first project of the Math Magicians application. I set up the environment and tools needed to develop a React application. In the following projects, I will develop the actual application.
+
+
+## Built With
+- GitFlow
+- React
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Web browser
+- Code editor
+
+### Setup
+```bash
+git clone (git@github.com:BouaikIlham/Math-Magicians.git)
+cd Math-Magicians
+```
+
+## Show your support
+
+- Give a star if you like this project!
+
+## Authors
+
+-  GitHub : [@bouaik ilham](https://github.com/BouaikIlham)
+- Twitter: [@IlhamBouaik](https://twitter.com/IlhamBouaik)
+- LinkedIn: [@BouaikIlham](https://www.linkedin.com/in/bouaik-ilham-478478230/)
+
+## Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] Please update your Readme, currently, some headings are missing such as  

``` ##  License ``` section with the MIT license at the end of your README file to make it more professional. I would also recommend you add this. 😄  please refer to this [template](https://github.com/microverseinc/readme-template).